tensorflow_app/views/import_graphdef.py

```diff
@@ -149,7 +150,8 @@ def import_graph_def(request):
                 return JsonResponse({'result': 'error', 'error': 'Invalid URL\n'+str(ex)})
         else:
             return JsonResponse({'result': 'error', 'error': 'No GraphDef model found'})
-
+        
+        tf.keras.backend.clear_session()
```

Comment:
Same issue as above, try using keras.backend instead. And since you have imported clear_session(), why not just directly call clear_session()? (Instead of tf.keras.backend.clear_session()).